Resources/public/js/views/ez-fieldeditview.js

```diff
@@ -67,8 +67,9 @@ YUI.add('ez-fieldeditview', function (Y) {
          * @return {eZ.FieldEditView}
          */
         render: function () {
-            var defaultVariables = {
-                    fieldDefinition: this.get('fieldDefinition'),
+            var def = this.get('fieldDefinition'),
+                defaultVariables = {
```

Comment:
the issue was only about field views but I guess it's OK to do that on edit views as well.